cmd/ike/cmd/develop.go

```diff
@@ -77,6 +77,7 @@ func NewDevelopCmd() *cobra.Command {
 	}
 	developCmd.Flags().StringP("method", "m", "inject-tcp", "telepresence proxying mode - see https://www.telepresence.io/reference/methods")
 	developCmd.Flags().StringP("session", "s", "", "create or join an existing session")
+	developCmd.Flags().StringP("route", "", "", "specify traffic route options in the format of type:name=value")
```

Comment:
Is there any default value if not specified? If so maybe worth mentioning it in the help text.

Reply:
It defaults on the Server SIde: https://github.com/aslakknutsen/istio-workspace/pull/99/files#diff-41947331b7e282077cf9be05075567fdR66

Reply:
Done in de3556f